src/redux/categories/categoriesSlice.js

```diff
@@ -2,9 +2,9 @@ import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
   categories: [
-    { id: '1', category: 'Adventure' },
-    { id: '2', category: 'Action' },
-    { id: '3', category: 'Romance' },
+    // { id: '1', category: 'Fiction' },
+    { id: '1', category: 'Nonfiction' },
+    { id: '2', category: 'Adventure' },
   ],
 };
```

Comment:
[ Optional Changes ]

- [ ] In your initial state here, you could pass the string "Under construction" and not the two categories of objects. Ideally, it's just that string that's required. Your redux is working as expected and hence, I can't ask for changes but I highly recommend you pass one object with the text "Under construction" 👍🏽